### PR TITLE
The Witness: Local Laser Shuffle + Option Presets

### PR DESCRIPTION
**Pull request by NewSoupVi**
Added option presets.
One of the option presets relies on laser_shuffle: True with local lasers, which option presets don't support.
However, "shuffle_lasers: local" has actually been an option we have been considering adding for a while anyway, because it is a very popular setting.

Note: Every other place in the code should behave as expected in regards to the change in the laser shuffle option, as there is functionally no difference between "local" and "anywhere", and `if self.options.laser_shuffle` will resolve as True for both "local" and "anywhere" and as False for "off".

Tested:
Ran webhost locally and confirmed that the presets worked and no errors happened in the python console or browser console
Generated a couple of seeds to see whether local lasers works as intended

Btw: I know the comments don't show on the website, they're more for other ppl looking at the code

### CODE REVIEW

**Comment by NewSoupVi** [comment]
I would request a review from someone who plays The Witness on whether these presets make sense, before this PR is merged

**Comment by Exempt-Medic** [comment]
The printout "[Player]'s Witness world doesn't have any progression items. Please turn on Symbol Shuffle, Door Shuffle or Laser Shuffle." can seem a little strange with `shuffle_lasers: local` given that there _are_ progression items and laser shuffle isn't exactly "off". The printout also appears in a single-player world with `shuffle_lasers: local` but _not_ with `shuffle_lasers: anywhere` which seem unusual/unintended.

**Comment by Silvris** [comment]
> The printout "[Player]'s Witness world doesn't have any progression items. Please turn on Symbol Shuffle, Door Shuffle or Laser Shuffle." can seem a little strange with `shuffle_lasers: local` given that there _are_ progression items and laser shuffle isn't exactly "off". The printout also appears in a single-player world with `shuffle_lasers: local` but _not_ with `shuffle_lasers: anywhere` which seem unusual/unintended.

Maybe be more specific in the message stating you need items to place in other worlds? If we allow laser_shuffle: local to count for that, the Witness player will have all of their progression items be forced into their own world, which is effectively the same as off.

**Review by Silvris — APPROVED** [review]
Changes LGTM, not a Witness player so can't speak to the quality of the presets.

**Comment by NewSoupVi** [comment]
Alright, I changed those messages to be a bit more sensible and specific and accurate :)

**Review by Exempt-Medic — APPROVED** [review]
Changes all look good to me, including the presets

**Comment by NewSoupVi** [comment]
This PR will break if merged together with https://github.com/ArchipelagoMW/Archipelago/pull/2631

**Comment by NewSoupVi** [comment]
This PR will break if https://github.com/ArchipelagoMW/Archipelago/pull/2631 is merged. @The core dev reading this: Only hit merge on one of them, then ping me on the other one

**Comment by NewSoupVi** [comment]
This PR depends on https://github.com/ArchipelagoMW/Archipelago/pull/2631

File: worlds/witness/__init__.py
Comment (Exempt-Medic) [inline worlds/witness/__init__.py:130]:
```suggestion
            self.options.local_items.value |= self.item_name_groups["Lasers"]
```

**Review by Berserker66 — APPROVED** [review]

**Pull request merged**
